cfme/tests/automate/test_git_import.py

```diff
@@ -79,27 +80,29 @@ def test_automate_git_domain_displayed_in_service(appliance, imported_domain):
         initialEstimate: 1/20h
         tags: automate
     """
+    url = cfme_data.ansible_links.playbook_repositories.automate_domain
+    repo = AutomateGitRepository(url=url, verify_ssl=True, appliance=appliance)
+    imported_domain = repo.import_domain_from(branch="origin/master")
     collection = appliance.collections.catalog_items
     cat_item = collection.instantiate(collection.GENERIC, "test")
     view = navigate_to(cat_item, "Add")
-    view.provisioning_entry_point.click()
-    view.modal.tree.click_path(
+    path = (
         "Datastore",
         "{0} ({1}) ({0}) (Locked)".format(imported_domain.name, "origin/master"),
         "Service",
-        "Provisioning",
+        "Generic",
         "StateMachines",
-        "ServiceProvision_Template",
-        "CatalogItemInitialization"
+        "GenericLifecycle",
+        "provision"
+    )
+    view.provisioning_entry_point.fill(path, include_domain=True)
+    assert view.provisioning_entry_point.value == (
+        "/{}".format("/".join([imported_domain.name, path[2], path[3], path[4], path[5], path[6]]))
```

Comment:
looks a bit not accurate. I guess the following would look better:
`[some_value, *path[1:]]`

Reply:
and I would give a shot to list/tuple comparison instead. I mean you can split entry point instead of joining these values

Reply:
- We need to include domain while selecting entry point in catalog item.
- This is useful if we have same tree but different domains available. So in this case we need to select domain prefix.


![screen_cfme](https://user-images.githubusercontent.com/25399590/62133794-0dd45500-b2fd-11e9-9ffe-91b04d9bcaba.png)